### PR TITLE
WAZO-1809: add do not disturb to user presence

### PR DESCRIPTION
**Pull request by bloom1**
Depends-On: https://github.com/wazo-platform/wazo-confd/pull/136

### CODE REVIEW

File: alembic/versions/6ba500c45fcc_add_dnd_to_presence.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = '6ba500c45fcc'
+down_revision = '543bc8a0045f'
+
+def upgrade():
+    op.add_column(
+        'chatd_user', sa.Column('do_not_disturb', sa.Boolean, nullable=False, server_default='0')
```
Comment (fblackburn1) [inline alembic/versions/6ba500c45fcc_add_dnd_to_presence.py:17]:
```suggestion
        'chatd_user', sa.Column('do_not_disturb', sa.Boolean, nullable=False, server_default='false')
```

File: wazo_chatd/database/models.py [hunk 2]
```diff
@@ -45,6 +45,7 @@ class User(Base):
         nullable=False,
     )
     status = Column(Text())
+    do_not_disturb = Column(Boolean(), nullable=False, default=False)
```
Comment (fblackburn1) [inline wazo_chatd/database/models.py:48]: Your model is not the same as the migration script
```suggestion
    do_not_disturb = Column(Boolean(), nullable=False, server_default='false')
```

File: integration_tests/suite/test_event_handler.py [hunk 3]
```diff
@@ -515,3 +515,30 @@ def channel_unheld():
                 )
             ),
         )
+
+    @fixtures.db.user()
```
Comment (fblackburn1) [inline integration_tests/suite/test_event_handler.py:519]: you should explicitly set dnd to False when it's what you test
```suggestion
    @fixtures.db.user(do_not_disturb=False)
```

**Comment by sboily** [comment]
I'm agree with that.

**Pull request merged**